config/webpack.client.js

```diff
@@ -150,7 +155,10 @@ module.exports = (env, argv) => {
         },
       },
       minimize: true,
-      minimizer: [new CssMinimizerPlugin(), new TerserPlugin()],
+      minimizer: [
+        new CssMinimizerPlugin(),
+        isProduction && new TerserPlugin(),
```

Comment:
Is there really no other workaround than to limit the JavaScript minimizer to only production? :( 

By the way, since we're on the topic of speed improvement, have you looked into the use of [multi-process parallel](https://webpack.js.org/plugins/terser-webpack-plugin/#parallel)? 🚅

Reply:
We could probably enable this for development (not the issue here), but it won't matter much tbh. And I/most prefer faster build time while developing rather than optimizing.

Reply:
Have not looked into the parallel option :thinking:. Could try it out and see if it makes a difference.